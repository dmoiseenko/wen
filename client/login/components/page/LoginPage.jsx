import React from 'react';

import Aside from '../aside/Aside';
import Logo from '../../../common/components/Logo';
import LoginFormContainer from '../form/LoginForm.container';


export default function LoginPage() {
  return (
    <div className="login-wrapper columns">
      <Aside />
      <div className="column is-4">
        <section className="hero is-fullheight">
          <div className="hero-head">
            <div className="section has-text-centered">
              <Logo large />
            </div>
          </div>
          <div className="hero-body">
            <div className="container">
              <div className="columns">
                <div className="column is-8 is-offset-2">
                  <LoginFormContainer />
                  <div className="section forgot-password">
                    <p className="has-text-centered">
                      <a href="https://moiden.com/dmoiseenko/wen/wikis/demo-credentials">Need help?</a>
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </section>
      </div>
    </div>
  );
}
