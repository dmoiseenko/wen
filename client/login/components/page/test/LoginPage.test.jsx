import React from 'react';
import { shallow } from 'enzyme';

import LoginPage from '../LoginPage';


it('should render', () => {
  const props = {};

  const tree = shallow(
    <LoginPage {...props} />
  );

  expect(tree.node).toMatchSnapshot();
});
