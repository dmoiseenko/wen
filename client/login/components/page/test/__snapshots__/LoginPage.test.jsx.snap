// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<div
  className="login-wrapper columns"
>
  <Aside />
  <div
    className="column is-4"
  >
    <section
      className="hero is-fullheight"
    >
      <div
        className="hero-head"
      >
        <div
          className="section has-text-centered"
        >
          <Logo
            large={true}
          />
        </div>
      </div>
      <div
        className="hero-body"
      >
        <div
          className="container"
        >
          <div
            className="columns"
          >
            <div
              className="column is-8 is-offset-2"
            >
              <Connect(ReduxForm) />
              <div
                className="section forgot-password"
              >
                <p
                  className="has-text-centered"
                >
                  <a
                    href="https://moiden.com/dmoiseenko/wen/wikis/demo-credentials"
                  >
                    Need help?
                  </a>
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
  </div>
</div>
`;
